### ISSUE TITLE
Can't seem to log on Support Notifications for Zendesk

### ISSUE THREAD

**Issue by SRTM01** [danabr/SupportNotifications#14]
Hi there,

I can't seem to log on my Zendesk account in Support Notifications. I can save my details. But when I click 'Test' the message pops-up 'Failed to authenticate!'.

Can somebody help me out?

**Comment by danabr** [comment]
Hi,
Support Notifications has not been actively developed for the last 4 years. I no longer have access to ZenDesk myself, and have not used this software for a very long time. I am not sure it works at all anymore.

I would advise you to look if there is an official chrome extension for ZenDesk instead.

With that  said, when looking at the code, it seems like you would get this message if either the password you provided is invalid, or it can't find the company id. Check if you can go directly to this URL in your browser (replace <company_id> with your company id): "https://<company_id>.zendesk.com/views".

**Comment by danabr** [comment]
Here is a chrome extension that seems to be able to provide similar functionality as SupportNotifications once did: https://www.integromat.com/en/integration/959-desktop-notifications-in-google-chrome-for-zendesk-tickets

I have not tested it myself, but it does look promising.

**Comment by SRTM01** [comment]
Hi Danabr,

Thanks for your response. I filled in the company ID, didnt work. I filled in our-company-id.zendesk.com, didnt work. :/. Even tried starting with https://. Nothing.

Allready tried Integromat, it isn't that simple and definitely not ideal. It worked, but in a while it's a paid feature. Right now I've installed notifications being pushed in Slack. Notifications work great, don't 'stick' on your screen though. They come and go. So not like Support Notifications does.

But SN should work right even without any development. Am I filling in the form incorrectly? Can we maybe take this off this website/communicate through email? So I can send you screen shots with the filled in fields.

Awaiting for your respons.
